src/components/presentational/CalendarDay/CalendarDay.css

```diff
@@ -19,13 +23,28 @@
 }
 
 .mdhui-calendar-day-value {
+    background: #DBDBDB;
     height: 40px;
     width: 40px;
     line-height: 40px;
     border-radius: 50%;
     text-align: center;
     vertical-align: middle;
-    margin: -5px auto 0;
+    margin: -10px auto 0;
```

Comment:
Curious if there is a reason why this top margin is -10 vs -12px?

Reply:
The `-10px` is the default without a border.  When the `2px` border is added by the two style classes below, the margin needs to be `-12px` to get the alignment to be correct.